queue/queue_test.go

```diff
@@ -411,6 +411,8 @@ func TestQueueSmoke(t *testing.T) {
 				}
 
 				t.Run(runner.Name+"Pool", func(t *testing.T) {
+					var testRetryOnce, testWaitUntilOnce, testDispatchBeforeOnce, testMaxTimeOnce, testScopesOnce, testApplyScopesOnEnqueueOnce sync.Once
```

Comment:
nit: can you make this more readable by doing:
```
var (
        testRetryOnce sync.Once
        testWaitUntilOnce
        ...
)
```

Reply:
Done.